wai-handler-hal.cabal

```
@@ -50,13 +50,13 @@ common deps
   build-depends:
     , base                  >=4.12      && <4.19
     , base64-bytestring     >=1.0.0.0   && <1.3
-    , bytestring            >=0.10.8    && <0.12.1
+    , bytestring            >=0.10.8    && <0.12.2
```

```suggestion
    , bytestring            >=0.10.8    && <0.13
```
If things are working with an `0.12.x` release, PVP says there won't be any breaking changes until `0.13`